components/PageCard.tsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState } from 'react';
+import { Flex, Text } from '@100mslive/react-ui';
+
+import { DynamicIcon } from './Callout';
+import { ArrowRightIcon } from '@100mslive/react-icons';
+
+const PageCard = ({
+    showCta = true,
```

Comment:
Here, the prop can be "CTA" with "Read more" as the default. If it is not needed, it can be specified as "" by the author